config/common-config.yaml

```diff
@@ -583,10 +583,9 @@ ingressNginx:
     enablepublishService: false
 
     ## Enable validation of Nginx annotations on Ingress resources.
-    ## This is kept false by default due to the maturity of the feature,
-    ## e.g. lack of documentation and disruptive bugs.
+    ## Only does runtime validation and produces error logs for invalid ingress annotations.
     ## TODO: Revisit this feature in future upgrade
-    enableAnnotationValidations: false
+    enableAnnotationValidations: true
```

Comment:
Could you also update the (perhaps somewhat redundant) `... enableAnnotationValidations.default = false` in the schema?